packages/genesys-spark-components/src/components/stable/gux-form-field/gux-form-field.service.ts

```diff
@@ -83,6 +83,11 @@ export function validateFormIds(
     error.setAttribute('id', errorId);
     describedByIds.push(errorId);
 
+    if (input instanceof HTMLInputElement) {
```

Comment:
Why this restriction?

Reply:
Some slotted 'inputs' are actually web components, `aria-invalid` can only be applied to an` <input>`.
Eg. without the condition in `gux-form-field-phone`, `aria-invalid` is added to the slotted `gux-phone-input-beta`.